examples/query/react/basic/package.json

```diff
@@ -24,12 +26,13 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "test": "react-scripts test"
```

Comment:
```suggestion
    "test": "react-scripts test --runInBand"
```

Reply:
`msw` recommends using `runInBand` to prevent the possibility of overlapping handlers when running in parallel. I'd rather show this as the default approach upfront at this time.